CMakeLists.txt

```diff
@@ -36,29 +35,29 @@ IF (NOT DEFINED RESTLIB_MODEL)
     set(RESTLIB_MODEL OFF)
 ENDIF ()
 
-#auto schema evolution for ROOT
+# Auto schema evolution for ROOT
```

Comment:
There is a parsing tool you are using to reformat this?

Reply:
> There is a parsing tool you are using to reformat this?

No, I did it by hand while I read the code. I think it's more legible this way, don't you agree?

Reply:
Yes, looks more tidy